compact/tracking/tof_endcap.xml

```diff
@@ -107,7 +107,7 @@
     <comment> 1 um padding to not have layer and module touch (ACTS requirement) </comment>
     <constant name="TOFEndcapLayer_thickness"   value="TOFEndcapMod_thickness + 2 * TOFEndcapMod_dz + 1 * um" />
 
-    <constant name="ForwardTOFMod_rmin"         value="ForwardTOFRegion_minR+5.0*mm" />
+    <constant name="ForwardTOFMod_rmin"         value="ForwardTOFRegion_minR-1*cm" />
```

Comment:
What is this 1 cm? Does it violate some detector envelope? Is it possible to have a named constant for it?

Reply:
That is correct. Volumes overlap if the minR is too large. Now that I think about it, I can just reduce ForwardTOFRegion_minR by 1 cm. checkOverlap shows 0 overlap when I do that. Let me push the updated version.

Reply:
The pull request is updated.

Reply:
Makes sense to me. Thanks!